ast.go

```diff
@@ -145,7 +145,12 @@ func (fa *ASTFunc) generateParameters() []ast.Expr {
 		if p.Type.IsSlice && !p.Type.IsReturnArgument {
 			hasDeclaration = true
 
-			fa.addGoToCSliceConversion(p.Name, p.Type)
+			
+			if _, ok := p.Heuristics["pass_c"]; !ok {
```

Comment:
You might kill me for this but I do not think that we should add a "Heuristics" map to any struct:
- Your intention for the maps was probably to make future similar changes easier but the future type interaction is very limited even later, but even then I think that it is easier and type-safer to add a flag.
- The only map that is currently used is of the FunctionParameter, so the other maps are not needed at all right now.
- Using keys makes it harder to safely use an API -> no compile errors

So I would be proposing to add a flag to the FunctionParameter struct BUT I am wondering. Shouldn't this be the default anyhow?